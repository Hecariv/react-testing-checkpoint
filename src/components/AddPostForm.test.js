import React from 'react';
import renderer from "react-test-renderer"

import AddPostForm from "./AddPostForm";



test("renders the same way every time", () => {

    const tree = renderer.create(<AddPostForm.WrappedComponent/>).toJSON();
    expect(tree).toMatchSnapshot();

})