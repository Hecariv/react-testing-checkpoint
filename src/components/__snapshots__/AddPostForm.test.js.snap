// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the same way every time 1`] = `
<div
  className="row"
>
  <div
    className="col-sm-10"
  >
    <form
      className=""
      onSubmit={[Function]}
    >
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="title-field"
        >
          Title
        </label>
        <input
          className="form-control"
          id="title-field"
          name="title"
          onChange={[Function]}
          required={true}
          type="text"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="body-field"
        >
          Body
        </label>
        <input
          className="form-control"
          id="body-field"
          name="body"
          onChange={[Function]}
          required={true}
          type="text"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="author-field"
        >
          Author
        </label>
        <input
          className="form-control"
          id="author-field"
          name="author"
          onChange={[Function]}
          required={true}
          type="text"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="image-field"
        >
          Image URL
        </label>
        <input
          className="form-control"
          htmlFor="ulr"
          id="image-field"
          name="image"
          onChange={[Function]}
          placeholder="https://example.com"
          required={true}
          type="url"
        />
      </div>
      <button
        aria-label={null}
        className="btn btn-secondary disabled"
        disabled={true}
        onClick={[Function]}
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
